index.html

```diff
@@ -51,9 +53,169 @@ <h1 class="header-title">Hey there. <br>I am Alex.</h1>
 
     </section>
 
+    <!-- Recent Work section -->
+    <section class="recent-work">
+        <div class="recent-work-container">
+            <div class="section-title title">
+                <div class="title-container">
+                    <h1>My Recent <br> Works</h1>
+                    <span class="section-indicator"></span>
+                </div>
```

Comment:
- [ ] Great job implementing the design template but there are some things you need to fix. As a standard practice, only the main title is required to be wrapped in an `<h1>` tag. Other titles can be wrapped with any of the `hx ` tag except the h1. This is so for semantic and conventional reasons. it also promotes your website's search optimization and accessibility. So kindly fix this.